src/modules/mc_pos_control/PositionControl/PositionControl.hpp

```diff
@@ -51,6 +51,9 @@ struct PositionControlStates {
 	float yaw;
 };
 
+#define POSCONTROL_HOVER_THRUST_MIN 0.05f	// The minimum level of hover thrust for constraining the hover thrust estimate
+#define POSCONTROL_HOVER_THRUST_MAX 0.9f	// The maximum level of hover thrust for constraining the hover thrust estimate
```

Comment:
Probably the constraints (min/max) should be pulled in from `MPC_THR_MIN` and `MPC_THR_MAX` with checking for sanity not crazy small and not crazy large?